src/xdocs/config_filters.xml

```diff
@@ -114,6 +114,13 @@
             Note that the suppression comment should be put before the violation. You can use more
             than one suppression comment each on separate line.
           </p>
+         <p>
+           Limitation: This filter may only be specified within the TreeWalker module
+           (<code>&lt;module name=&quot;TreeWalker&quot;/&gt;</code>) and only
+           applies to checks which are also defined within this module. 
+           To filter non-TreeWalker checks like <code>RegexpSingleline</code>, 
+           the <code>SuppressWithPlainTextCommentFilter</code> filter is an alternative.
```

Comment:
`filter is an alternative.` => `or similar filter must be used.`

You make it sound like this filter works, but you can also use this other filter, which isn't the case. It must be one or the other.